packages/frontend-2/lib/viewer/composables/serialization.ts

```diff
@@ -115,7 +115,9 @@ export function useStateSerialization() {
           : null,
         lightConfig: { ...state.ui.lightConfig.value },
         explodeFactor: state.ui.explodeFactor.value,
-        selection: state.ui.selection.value?.toArray() || null
+        selection: state.ui.selection.value?.toArray() || null,
+        measurementOptions: state.ui.measurementOptions.value,
```

Comment:
Why not just?

```
measurement: {
  options: Ref<...>,
  enabled: Ref<...>
}
```

Otherwise 1) they're not grouped together in a single object and 2) they repeat the same redundant "measurement" prefix twice